cfme/tests/services/test_rest_services.py

```diff
@@ -1590,7 +1594,7 @@ def test_create_second_cart(self, request, appliance, cart):
 
     @pytest.mark.tier(3)
     # not testing on version < 5.9 due to BZ1493785 that was fixed only in 5.9
-    @pytest.mark.uncollectif(lambda: store.current_appliance.version < '5.9')
+    @pytest.mark.uncollectif(lambda appliance: appliance.version < '5.9')
```

Comment:
Required: this can be removed

Reply:
Right, versions < 5.9 are no longer needed and since there's lot of uncollectifs for these + tests valid only for versions < 5.9 (blueprints etc.), I'll remove these in separate PR.